apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -475,7 +498,7 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
         }
 
         data.Operations[0].value = value;
-        updateProfileInfo(data).then((response) => {
+        updateProfileInfo(data).then((response: any) => {
```

Comment:
Shall we avoid using `any`? Check other places as well.

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3763/commits/25d7975004f07604b9d6adb6dd1e18041b8ef502